source/releasenotes/2025-01-15-php-53-55-eol.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "PHP 5.3 and 5.5 no longer available starting February 25, 2025"
+published_date: "2025-01-15"
+categories: [infrastructure, security, action-required]
+---
+As part of our continued effort to protect and secure customer sites, the Pantheon platform will no longer offer PHP versions 5.3 and 5.5 as of February 25, 2025.
+
+PHP 5.5 was declared end-of-life (EOL) by [the PHP Foundation](https://www.php.net/supported-versions.php) on July 21, 2016, more than eight years ago. EOL software does not receive security or feature updates, and could expose sites to attack if any vulnerabilities or exploits are discovered.
+
+We will soon provide long term support (LTS) security coverage for all available PHP versions on the platform.
```

Comment:
"We will soon provide long term support (LTS) security coverage for the remaining EOL versions of PHP on the platform." ? @jazzsequence

Reply:
👍 Works for me